SPECS/python-setuptools/python-setuptools.spec

```diff
@@ -19,14 +19,15 @@ Source0:        https://pypi.org/packages/source/s/setuptools/setuptools-%{versi
 %package -n python3-setuptools
 Summary:        %{summary}
 
-Requires:       python3-devel
 # Early builds of Azure Linux 3.0 included python3-setuptools with the python3.spec. Obsolete to prevent build conflicts.
 Obsoletes:      python3-setuptools <= 3.9.14
 BuildArch:      noarch
 
 # Note: these build requirements are only for the non-toolchain build environment (since they are already available in the toolchain environment)
-#BuildRequires:  python3-pip
-#BuildRequires:  python3-wheel
+BuildRequires:  python3-devel
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+Requires:       python3
```

Comment:
Even this is not needed - `python3-setuptools` already requires `python(abi) = 3.12`, which translates to `python3`.

Reply:
thanks. I don't think it hurts to add it though - so will keep as-is